Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/FileUtil.java

```diff
@@ -108,18 +109,26 @@ public static boolean isNullOrEmpty(final String name) {
 	}
 
     /**
-     * Utility method to zip a file.
-     * @param inputStream
-     * @param fileName
-     * @return zip file
-     * @throws IOException
+     * Utility method to zip the given source file to the destination file.
+     * @param sourceFile source file
+     * @param targetZipFile ZIP file that will be created or overwritten
      */
-    public static File zipFile(InputStream inputStream, String fileName) throws IOException {
-        final File tempZip = File.createTempFile(String.valueOf(new Date().getTime()), ".zip");
-        final FileOutputStream fos = new FileOutputStream(tempZip);
+    public static void zipFile(final @NotNull File sourceFile, final @NotNull File targetZipFile) throws Exception {
+        if (!sourceFile.exists()) {
+            throw new Exception("The source file to zip does not exist.");
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/io/FileNotFoundException.html

Reply:
Anyway I think we can do it later, just don't forget it.

Reply:
Track it in this issue https://github.com/Microsoft/azure-tools-for-java/issues/1983